### ISSUE TITLE
[P2][Sourcing][Accessibility][Tree table] : Tree table expand all / collapse all change state is not conveyed for table rows by screenreader

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#11374]
#### Is this a bug, enhancement, or feature request?

Tree table expand all / collapse all change state is not conveyed from screenreader
The state of the buttons ‘Expand all’ and ‘Collapse all’ is not read to the users. Here on activating these buttons the tree levels of the table gets expanded/collapsed, but this change in state is not conveyed.

#### Describe your proposal.
State, properties of all UI elements should be conveyed correctly to the users when accessed.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core / platform: 0.43.26

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Expand all the rows and notice that nothing is being conveyed to users that rows are being expanded
similarly for collapse all button

#### Please provide relevant source code (if applicable).
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#tree

#### Please provide stackblitz example(s).
replicable on https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#tree

#### Is there anything else we should know?
Reported by ATL team

**Comment by mikerodonnell89** [comment]
I'm hesitant to add this to the library itself for 2 reasons:

1. These are buttons, not toggle buttons, and I don't think it is standard practice for the screenreader to announce the user has clicked a button after they do so. 
2. Expanding/collapsing items will often require multiple API calls which may pass or fail, I think it's best for the app developer to decide when they want to make this announcement.

**Comment by I543348** [comment]
Thanks @mikerodonnell89 
Probably @nikolay-kolarov could you pls provide your inputs here?
Thanks

**Comment by nikolay-kolarov** [comment]
Hello @I543348,

I personally do not see a need to add such a message in this case, but you can double check this with UX/acc - for me it is logical that after pressing an expand all button, the results will be expanded and when the user goes down they also will find out the state with the screen reader. And I do not see it so much as a change of state, but rather as a change in visualization. So I do not think you need additional announcement.

Best Regards,
Nikolay
